### PR TITLE
simply tts to prevent it freezes the app

### PR DESCRIPTION
**Pull request by nyagami**
Also this feature uses paragraphs as breakpoints.

### CODE REVIEW

File: src/hooks/useTextToSpeech.js [hunk 1]
```diff
@@ -1,100 +1,40 @@
...
   useEffect(() => {
-    Tts.addEventListener('tts-start', () => setTtsStatus('start'));
-    Tts.addEventListener('tts-progress', event => {
-      setTtsStatus('progress');
-      setTtsPosition(event);
-    });
     Tts.addEventListener('tts-finish', () => {
-      setTtsStatus('finish');
-      markChapterAdRead();
+      if (ttsPosition == sentences.length - 1) {
```
Comment (rajarsheechatterjee) [inline src/hooks/useTextToSpeech.js:9]: Shouldn't it be `ttsPosition.current`?

Comment (nyagami) [reply src/hooks/useTextToSpeech.js:9]: ah yes, my bad :>

**Review by rajarsheechatterjee — APPROVED** [review]

**Comment by sjain3097** [comment]
When will this be released for the apk, the app is almost unusable for me without this fix

**Comment by rajarshee-harness** [comment]
You can download it from [here](https://github.com/LNReader/lnreader/suites/10873325295/artifacts/548562285).

**Comment by sjain3097** [comment]
Working perfectly. Thanks

**Pull request merged**